docs/Installing.asciidoc

```diff
@@ -18,34 +18,70 @@ assumed that the reader is already familiar with openQA and has already read the
 Starter Guide, available at the
 https://github.com/os-autoinst/openQA[official repository].
 
+
```

Comment:
why?

Reply:
No special reason, I did it for me to add readability. I can remove it.